test/t/index/test_dump_and_load_index.cpp

```diff
@@ -0,0 +1,45 @@
+#include "catch.hpp"
+
+#include <osmium/index/detail/tmpfile.hpp>
+#include <osmium/index/map/dense_file_array.hpp>
+#include <osmium/index/map/dense_mmap_array.hpp>
+#include <osmium/osm/location.hpp>
+#include <osmium/osm/types.hpp>
+
+using dense_mmap_array_t = osmium::index::map::DenseMmapArray<osmium::unsigned_object_id_type, osmium::Location>;
```

Comment:
Please don't use types with `_t` names. I know this is quite common but strictly speaking all names ending in `_t` are reserved by the C++ standard (even those in other names spaces).

Reply:
`_t` removed